modules/shadertools/test/lib/assemble-shaders.spec.js

```diff
@@ -28,6 +29,117 @@ void main(void) {
 }
 `;
 
+const VS_GLSL_300_2 = `\
```

Comment:
Add a test case for in/out qualifier in function params? e.g. https://github.com/uber/deck.gl/blob/master/modules/core/src/shaderlib/project32/project32.js#L25